controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -93,6 +104,16 @@ public void createStream(String scope, String streamName, int minNumSegments, Du
         createStreamLatency.reportSuccessValue(latency.toMillis());
     }
 
+    /**
+     * This method increments the global counter of Scope creations and reports the latency of the operation.
+     *
+     * @param latency           Latency of the createStream operation.
+     */
+    public void createScope(Duration latency) {
+        DYNAMIC_LOGGER.incCounterValue(CREATE_SCOPE, 1);
```

Comment:
this is counting how many scopes this controller instance has created so far..

Reply:
The values from multiple Controllers will be aggregated at the InfluxDB level when collecting the metrics.